resources/views/controllers/gatwick.blade.php

```diff
@@ -0,0 +1,133 @@
+                    <br>
+                    <h4>Step Two</h4>
+                    You will be given access to the 'Gatwick ADC | S1 Endorsement' course. This Moodle course covers Gatwick specific procedures, radiotelephony, and local flight planning restrictions.
+                    There is a quiz at the end of the course with a pass mark of 80% - you must pass this quiz to proceed.
+                    If you do not pass the quiz on your first attempt, there is a study period of seven days for you to review the Moodle course and improve your knowledge before you try again.<br>
+                    When you have passed the quiz at the end of the Moodle course, you will be prompted to submit another ticket to ATC TRAINING.<br>
+                    <br>
+                    <h4>Step Three</h4>
+                    Your training now moves onto the live network. One of our S1 mentors will take you onto EGKK_GND for a 1h30 session, and offer you hints and tips as you control, answering any questions you may have.
+                    This is not a test, and you will not pass or fail, rather it is an opportunity for you to practically apply the skills and knowledge which you have learned through completing the Moodle course.<br>
+                    Once you have completed the session, your mentor will submit a request for you to be added to the special endorsement list. Once this has been processed, you will be able to control EGKK_DEL and EGKK_GND on the live network.
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-md-4">
+            <div class="panel panel-ukblue">
+                <div class="panel-heading"><i class="glyphicon glyphicon-info-sign"></i> &thinsp; Group One Controlling</div>
+                <div class="panel-body">
+                    Control a total of <strong>10 hours</strong> across the following positions within the last <strong>3 months.</strong>
+                    <ul>
+                        <li>Manchester (EGCC)</li>
+                        <li>Edinburgh (EGPH)</li>
+                        <li>Stansted (EGSS)</li>
+                        <li>Liverpool (EGGP)</li>
+                    </ul>
+                    <div class="progress">
+                        @if($groupone == 0)
+                            <div class="progress-bar" role="progressbar" style="width: 0%" aria-valuemin="0" aria-valuemax="10"></div>
+                        @elseif($groupone > 10)
+                            <div class="progress-bar progress-bar-success" role="progressbar" style="width: 100%" aria-valuemin="0" aria-valuemax="10">{{ $groupone }} Hours</div>
+                        @else
+                            <div class="progress-bar" role="progressbar" style="width: {{ $groupone*10 }}%" aria-valuemin="0" aria-valuemax="10">{{ $groupone }} Hours</div>
```

Comment:
What is the point of checking if they have 0 hours? The progress pass will still be displayed with zero hours exactly the same way. (0*10=0)